saleor/graphql/checkout/mutations.py

```diff
@@ -435,6 +456,10 @@ class Meta:
         error_type_class = CheckoutError
         error_type_field = "checkout_errors"
 
+    @classmethod
+    def check_permissions(cls, context):
+        return context.user.is_authenticated
+
     @classmethod
     def perform_mutation(cls, _root, info, checkout_id):
```

Comment:
We should check if the currently logged in user is the current owner of that checkout session. A malicious user could disassociate all checkouts from their current owners by enumerating all possible checkout IDs.

Reply:
Added a check for that.